cdap-master/src/main/java/co/cask/cdap/master/startup/ConfigurationCheck.java

```diff
@@ -93,12 +93,9 @@ private void checkBindAddresses() {
     // check if service bind addresses are loopback addresses
     Set<String> bindAddressKeys = ImmutableSet.of(
       Constants.AppFabric.SERVER_ADDRESS,
-      Constants.Dataset.Executor.ADDRESS,
       Constants.Stream.ADDRESS,
```

Comment:
Not sure why they are in cdap-default.xml. The only thing I can think of is they can be changed in SDK.